starWarsQuotesRepository.py

```diff
@@ -42,15 +43,18 @@ def __getQuotes(self, trilogy: str = None) -> List[str]:
         return result
 
     def __processQuote(self, quote: str, input: str = None) -> str:
-        if not utils.isValidStr(input):
-            return quote
+        if not utils.isValidStr(quote):
+            return None
```

Comment:
I think we should delete these two lines now, as this should only ever happen if we have a bug somewhere else.